apps/code-examples/project.json

```diff
@@ -102,7 +102,20 @@
           "libs/components/theme/src/lib/styles/themes/modern/styles.scss"
         ],
         "scripts": [],
-        "assets": []
+        "assets": [],
+        "inlineStyleLanguage": "scss",
+        "stylePreprocessorOptions": {
```

Comment:
Are these passed to `ng-packagr`? Do we need to include the `inlineStyleLanguage` and `includePaths` both in the `project.json` and the `ng-package.json`? Could we opt to only including it in one place?
See: https://github.com/blackbaud/skyux/blob/main/libs/components/a11y/ng-package.json

Reply:
These are passed to the [karma builder](https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/src/builders/karma/schema.json#L117), it receives its options differently than the `ng-packagr` builder, and I don't see a way to combine the two set of options.